completions/helm.completion.sh

```diff
@@ -0,0 +1,5 @@
+#! bash oh-my-bash.module
+if _omb_util_command_exists helm; then
+  source <(helm completion bash)
+fi
+
```

Comment:
Could you strip empty lines at the end of files?

Reply:
done